code/modules/vehicles/multitile/multitile_interaction.dm

```diff
@@ -20,9 +20,6 @@
 
 	// Are we trying to repair the frame?
 	if(iswelder(O) || HAS_TRAIT(O, TRAIT_TOOL_WRENCH))
-		if(!HAS_TRAIT(O, TRAIT_TOOL_BLOWTORCH))
```

Comment:
still needs to check if the blowtorch is weak/strong to prevent blowtorches without the trait from repairing vehicles